Apps/Patient/src/Models/PatientModelV1.cs

```diff
@@ -0,0 +1,91 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Patient.Models
+{
+    using System;
+    using System.Text.Json.Serialization;
+    using HealthGateway.Common.Models;
+
+    /// <summary>
+    /// V1 Model for patient data, required for V1 GetPatient Endpoint.
+    /// </summary>
+    public class PatientModelV1
```

Comment:
Wouldn't it better to name this to PatientModelV2 and have it pointed to GetPatientV2 instead?  Then you can leave GetPatient untoched.

Note: eventually once we have problem details exception handling resolved, the ClientRegistriesDelegete and PatientService in Patient are going to be removed and pointed at Common at which time, these changes would be applied there at that time.